.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
```

Comment:
Can use a build in .gitignore file
https://github.com/github/gitignore/blob/master/Python.gitignore
(might be missing some pycharm staff)